src/data/faq_de.json

```diff
@@ -551,12 +551,13 @@
                         ]
                     },
                     {
-                        "title": "Was passiert, wenn ich Bluetooth und bei Android auch die Standort-Dienste ausschalte?",
+                        "title": "Was passiert, wenn ich Bluetooth und bei Android die Standort-Dienste ausschalte?",
                         "anchor": "bluetooth_off",
                         "active": true,
                         "textblock": [
-                            "Dann können Begegnungen mit den anderen Smartphones, die die Corona-Warn-App laufen haben, nicht aufgezeichnet werden, weil keine Zufalls-IDs ausgetauscht werden. Haben Sie deswegen bitte immer Bluetooth und bei Android auch die Standort-Dienste aktiviert, wenn Sie in Kontakt mit anderen Leuten sind, zum Beispiel, wenn Sie Gäste bekommen oder Sie unterwegs sind.",
-                            "Bei Android müssen die Standort-Dienste nur aktiviert sein, die Corona-Warn-App verwendet den Standort nicht. Für alle anderen Apps können Sie die Standortverwendung ausschalten, wenn Sie das möchten: siehe <a href='#android_location' target='_blank' rel='noopener'>Meldungen zur Aktivierung der Standortverwendung</a>."
+                            "Dann können Begegnungen mit den anderen Geräten, die die Corona-Warn-App laufen haben, nicht aufgezeichnet werden, weil keine Zufalls-IDs ausgetauscht werden. Haben Sie deswegen bitte immer Bluetooth (und bei Android auch die Standort-Dienste) aktiviert, wenn Sie in Kontakt mit anderen Leuten sind, zum Beispiel, wenn Sie Gäste haben oder nach draußen gehen.",

```

Comment:
Perhaps instead of:
"Haben Sie deswegen ..."
alternative
"Halten Sie deswegen ..." ?

Reply:
"Halten Sie Bluetooth aktiviert" and "Haben Sie Bluetooth aktiviert" both don't sound great to me.

Suggestions:
* "Aktivieren Sie Bluetooth …"
* "Lassen Sie Bluetooth aktiviert …"
* "Stellen Sie sicher, dass Bluetooth immer aktiviert ist, wenn …"

Reply:
@fynngodau 
The above is an outdated comment. Check instead https://github.com/corona-warn-app/cwa-website/pull/1052/files and the line says

"Aktivieren Sie deswegen bitte immer Bluetooth (und bei Android bis zur Version 10 auch die Standort-Dienste), wenn Sie in Kontakt mit anderen Leuten sind, zum Beispiel, wenn Sie Gäste haben oder nach draußen gehen."

Reply:
Ah, sorry. I think you can mark this as resolved if it's fixed, can't you?

Reply:
I don't have the option to mark as resolved here. Sorry.